doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/baseboard-heaters.tex

```diff
@@ -130,19 +128,23 @@ \subsubsection{Simulation and Control}\label{simulation-and-control-2-000}
 {T_{w,out}} = 2{T_{w,avg}} - {T_{w,in}}
 \end{equation}
 
-where \({c_{p,w}}\) ~is the specific heat capacity of the water.
+where \({c_{p,w}}\) is the specific heat capacity the water (calculated based on the average water temperature).
```

Comment:
*of* the water

Reply:
@Myoldmopar Another good catch--this is now also fixed.  The changes you requested should now be up in GitHub.